BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -24,31 +24,33 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
         if (item.isFolder) {
             BOXFolder *folder = (BOXFolder *)item;
             if (folder.isExternallyOwned == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-external"
+                icon = [UIImage imageNamed:@"External"
```

Comment:
I see, we used to have one icon per file type, which will have many duplicates, so now we have one icon per group of file types (group will be like Audio, Video), no need for more granular types of icons.